tests/TestClassDatabaseChanges.php

```diff
@@ -213,7 +213,7 @@ public function testUninstall(array $createQueries)
 		// assert that as many tables have been removed as have been created
 		$uniqueCreateQueries = array_unique($createQueries);
 		$uniqueDropQueries = array_unique($this->_dropQueries);
-		$this->assertEquals(count($uniqueCreateQueries), count($uniqueDropQueries));
+		$this->assertGreaterThanOrEqual(count($uniqueCreateQueries), count($uniqueDropQueries));
```

Comment:
If there is a difference here, that means there's some cleanup code missing.

Reply:
This needs to be `assertEquals()`

Reply:
because, in TestClassAdminViewController run install DBChanges in the first so _createQueries = 0 and $uniqueDropQueries >0 to delete so use assertGreaterThanOrEqual,

Reply:
If the dbVersion option got reset to null, the amount of created and dropped tables should be the same, right?

Reply:
But that is only the case if `$dbVersion` is not `null` or `0`, correct? If `$dbVersion` was set to `null`, the amount of created and dropped tables has to be the same.